packages/framework/src/version.ts

```diff
@@ -1,3 +1,4 @@
 import packageJson from '../package.json';
 
 export const VERSION = packageJson.version;
```

Comment:
Suggest to rename this to be more explicit.

```suggestion
export const SDK_VERSION = packageJson.version;
```